client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/js/common.js

```diff
@@ -1,22 +1,24 @@
 const ECHOENDPOINT_URL = `http://${document.location.host}/echo`;
 
+function getTransportTypes() {
+    let transportTypes = [ signalR.TransportType.WebSockets ];
```

Comment:
The order does not matter but I wanted to keep the same order as in signalR.TransportType enum as well as what it was before.